spec/system/hiring_staff_can_sign_in_by_email_spec.rb

```diff
@@ -217,7 +217,6 @@
             freeze_time do
               expect { visit auth_email_choose_organisation_path(login_key: login_key.id) }
                 .to have_triggered_event(:publisher_sign_in_attempt)
-                .with_base_data(user_anonymised_publisher_id: anonymised_form_of(publisher.oid))
```

Comment:
Is this data not there anymore/is it in the data struct instead?

And can we make sure the data is there if it isn't so we keep the performance analysis wheels turning smoothly?